daemon/container_unix.go

```diff
@@ -242,6 +242,12 @@ func populateCommand(c *Container, env []string) error {
 		}
 	}
 
+	var wDevices []string
+	for _, weightDevice := range c.hostConfig.BlkioWeightDevice {
+		wDeivces = append(wDevices, fmt.Sprintf("%s:%s", weightDevice.path, weightDevice.weight))
+	}
+	weightDevice := strings.Join(wDevices, "\n")
```

Comment:
This seems wrong though, I've opened an issue in libcontainer since writing newline separated weight devices isn't working but each weight  device should be written one at a time